src/VK/CMakeLists.txt

```diff
@@ -1,6 +1,10 @@
 project (Framework_VK)
 
-add_compile_options(/MP)
+if(MSVC)
+  add_compile_options(/MP)
+else()
+add_compile_options(-M)
```

Comment:
Is it really what you intended? I wouldn't expect generating make rules here.

Reply:
There was `add_compile_options(/MP)` in the `src/DX12` which is suprising too.

Reply:
`add_compile_options(/MP)` gives me a build error in GCC 9
Passing the `-M` switch as a compile option seems to resolve it